gprofiler/profilers/java.py

```diff
@@ -1163,24 +1172,27 @@ def _check_hotspot_error(self, ap_proc: AsyncProfiledProcess) -> None:
             return
 
         contents = open(error_file).read()
-        m = VM_INFO_REGEX.search(contents)
-        vm_info = m[1] if m else ""
-        m = SIGINFO_REGEX.search(contents)
-        siginfo = m[1] if m else ""
-        m = PROBLEMATIC_FRAME_REGEX.search(contents)
-        problematic_frame = m[1] if m else ""
-        m = NATIVE_FRAMES_REGEX.search(contents)
-        native_frames = m[1] if m else ""
-        m = CONTAINER_INFO_REGEX.search(contents)
-        container_info = m[1] if m else ""
-        logger.warning(
-            f"Found Hotspot error log for pid {pid} at {error_file}:\n"
-            f"VM info: {vm_info}\n"
-            f"siginfo: {siginfo}\n"
-            f"Problematic frame: {problematic_frame}\n"
-            f"native frames:\n{native_frames}\n"
-            f"container info:\n{container_info}"
-        )
+        if self._java_full_hserr:
+            logger.warning(f"Found Hotspot error log for pid {pid} at {error_file}:\n{contents}")
```

Comment:
Additionally - let's improve the message to use logging extras to be consistent with other code, so for example
```
logger.warning("Found Hotspot error log" pid=pid, hs_err_file=error_file, hs_err=contents)
```
same for the other branch (there you can concatenate the excerpts into `contents` variable and then log it)